doc/guides/diagnostic-tooling-support-tiers.md

```diff
@@ -0,0 +1,115 @@
+   Node.js CI in order to qualify for a higher tier.
+   
+The choice of which tier a particular tool will be assigned to, will be a
+collaborative decision between Diagnostics WG and Release WG. Some of the
+criteria considered might be:
+
+* If the tool fits into a key cateogry as listed below.
+* Whether the tool is actively used by the community.
+* The Availability of alternatives.
+* Impact to the overall ecosystem if the tool is not working.
+* The availability of reliable test suite that can be integrated into our CI.
+* The availability of maintainer or community collaborator who will help
+  resolve issues when there are CI failures.
+* If the tool is maintained the Node.js foundation GitHub organization.
+ 
+The current categories of tools/APIs that fall under these Tiers are:
+
+* FDDC (F) - First failure data capture, Easy to consume initial diagnostic
+  information.
+* Tracing (T) - use of logging to provide information about execution flow.
+* Memory (M) - tools that provide additional information about memory 
+  used in the Heap or by native code.
+* Profiling (P) - tools that provide additional information about where
+  cpu cycles are being spent.
+* AsyncFlow (A) - tools that provide additional insight into asynchronous
+  execution flow.
+
+ 
+The following tools are currently assigned to Tiers as follows
+ 
+## Tier 1
+
+ | Tool Type | Tool Name                 | Regular Testing In Node.js CI | Integrated with Node.js | Target Tier |
+ |-----------|---------------------------|-------------------------------|-------------------------|-------------|
+ |           |                           |                               |                         |             |
+ 
+## Tier 2
+
+ | Tool Type | Tool Name                 | Regular Testing In Node.js CI | Integrated with Node.js | Target Tier |
+ |-----------|---------------------------|-------------------------------|-------------------------|-------------|
+ |           |                           |                               |                         |             |
+ 
+ 
+## Tier 3
+ 
+ | Tool Type | Tool Name                 | Regular Testing In Node.js CI | Integrated with Node.js | Target Tier |
+ |-----------|---------------------------|-------------------------------|-------------------------|-------------|
+ |           |                           |                               |                         |             |
+
+## Tier 4
+
+ | Tool Type | Tool Name                 | Regular Testing In Node.js CI | Integrated with Node.js | Target Tier |
+ |-----------|---------------------------|-------------------------------|-------------------------|-------------|
+ |           |                           |                               |                         |             |
+ 
+## Not yet classified
+ 
+ | Tool Type | Tool/API Name             | Regular Testing In Node.js CI | Integrated with Node.js | Target Tier |
```

Comment:
Should the second column header be the same in all tables (`Tool Name` vs `Tool/API Name`)?

Reply:
Yes, fixed